src/main/resources/jupyter/google_sign_in.js

```diff
@@ -1,14 +1,3 @@
-require.config({
```

Comment:
No longer using [RequireJS](https://requirejs.org/) to load the Google JS client [library](https://developers.google.com/api-client-library/javascript/start/start-js). JupyterLab doesn't include RequireJS because it doesn't interact nicely with node. (There is a node [library](https://requirejs.org/docs/node.html) to support RequireJS but it broke like 10 other things when I tried to install it.)